addons/crm/data/crm_lead_demo.xml

```diff
@@ -396,9 +396,6 @@ Andrew</p>]]></field>
             <field name="probability">30.0</field>
             <field name="partner_id" ref="base.res_partner_2"/>
             <field name="email_from">virginie@agrolait.com</field>
-            <field name="country_id" ref="base.be"/>
-            <field name="city">Wavre</field>
-            <field name="street">Rue de Namur 69</field>
             <field name="phone">+32 10 588 558</field>
```

Comment:
We can maybe edit the phone number as well? `+32` is a phone prefix for Belgium. We can maybe provide a prefix from the US to be more consistent?